### DISCUSSION TITLE
What preset does Renovate use if nothing is provided

### DISCUSSION THREAD

**Discussion by ViRb3** [renovatebot/renovate#25189] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### Question

If I have the following Renovate config:

```js
module.exports = {
  onboarding: false,
  requireConfig: "optional",
  repositories: [
    "foo/bar"
  ]
};
```

And the repository `foo/bar` does not contain any renovate config, Renovate still works. What preset does it use? Is this preset sane? It's not the `config:recommended` since a dependency dashboard is missing.

Thanks!

**Comment by rarkins** [comment]
It uses renovate's default values

**Reply by ViRb3** [reply]
Is there any place I can see these default values? And are they sane defaults, or discouraged? For example, I am curious what the default PR limit is. There are presets for [:prHourlyLimit1](https://docs.renovatebot.com/presets-default/#prhourlylimit1), 2, 4, None, but neither of these is documented as being default. Thanks!

**Reply by rarkins** [reply]
All are at docs.renovatebot.com

**Reply by ViRb3** [reply, marked as answer]
Found defaults listed in docs here: https://docs.renovatebot.com/configuration-options
Ref to source code: https://github.com/renovatebot/renovate/blob/main/lib/config/options/index.ts
